pep-0590.rst

```diff
@@ -64,12 +74,14 @@ Changes to the ``PyTypeObject`` struct
 --------------------------------------
 
 The unused slot ``printfunc tp_print`` is replaced with ``tp_vectorcall_offset``. It has the type ``Py_ssize_t``.
-A new ``tp_flags`` flag is added, ``Py_TPFLAGS_HAVE_VECTORCALL``,
+A new ``tp_flags`` flag is added, ``_Py_TPFLAGS_HAVE_VECTORCALL``,
 which must be set for any class that uses the vectorcall protocol.
 
-If ``Py_TPFLAGS_HAVE_VECTORCALL`` is set, then ``tp_vectorcall_offset`` must be a positive integer.
+If ``_Py_TPFLAGS_HAVE_VECTORCALL`` is set, then ``tp_vectorcall_offset`` must be a positive integer.
 It is the offset into the object of the vectorcall function pointer of type ``vectorcallfunc``.
-This pointer may be ``NULL``, in which case the behavior is the same as if ``Py_TPFLAGS_HAVE_VECTORCALL`` was not set.
+This pointer may be ``NULL``, in which case the behavior is the same as if ``_Py_TPFLAGS_HAVE_VECTORCALL`` was not set.
+
+If ``_Py_TPFLAGS_HAVE_VECTORCALL`` is not set, then ``tp_vectorcall_offset`` must be zero.
```

Comment:
This is actually new content. We can't make it a hard requirement because of backwards compatibility: it's perfectly legal for an extension type to have a non-NULL value for `tp_print`, which will become a non-zero value for `tp_vectorcall_offset`. Can we please not add this sentence here and defer to the documentation when we implement PEP 590?